crates/nu-command/src/filesystem/start.rs

```diff
@@ -130,3 +133,47 @@ impl Command for Start {
         ]
     }
 }
+
+fn open_path(
+    path: impl AsRef<std::ffi::OsStr>,
+    engine_state: &EngineState,
+    stack: &Stack,
+    span: Span,
+) -> Result<(), ShellError> {
+    try_commands(open::commands(path), engine_state, stack, span)
+}
+
+fn try_commands(
+    commands: Vec<std::process::Command>,
+    engine_state: &EngineState,
+    stack: &Stack,
+    span: Span,
+) -> Result<(), ShellError> {
+    let env_vars_str = env_to_strings(engine_state, stack)?;
+    commands
+        .into_iter()
+        .map(|mut cmd| {
+            let status = cmd
+                .envs(&env_vars_str)
+                .stdin(Stdio::null())
+                .stdout(Stdio::null())
+                .stderr(Stdio::null())
+                .status();
+            match status {
+                Ok(status) if status.success() => Ok(()),
+                Ok(status) => Err(format!("\nCommand {cmd:?} failed with {:?}", status)),
+                Err(err) => Err(format!("\nCommand {cmd:?} failed with {:?}", err)),
```

Comment:
Debug formatting is debug formatting and shouldn't necessarily be used for regular user output.